src/core/quicdef.h

```diff
@@ -453,9 +453,15 @@ CXPLAT_STATIC_ASSERT(
 #define CXPLAT_TLS_RESUMPTION_CLIENT_TICKET_VERSION      1
 
 //
-// By default the Version Negotiation Extension is disabled.
+// By default the Version Negotiation Extension is disabled
+// when preview features are disabled. Otherwise, it's enabled.
 //
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
```

Comment:
Internally we always enable preview features. I don't understand this.

Reply:
The goal here is that when that flag is present, version negotiation defaults to enabled (which it wasn't before). But if someone disables the preview features, we disable version negotiation by default (and since the API to change it is also removed when that flag is not present, there's no way to enable it). 
The goal is making it more complete when a 3rd party disables preview features

Reply:
This is an internal header and doesn't change based off the caller. It will always be enabled.

Reply:
oh, I thought it was a CMAKE build flag. My mistake. I'll just enable Version Negotiation by default then.

Reply:
Nope, its not a build flag at all. Preview features are always enabled for the build. Its just that they might not show up for the callers, unless they specify the define before including msquic.h